main.py

```diff
@@ -185,9 +185,77 @@ def calculate_date_range(start, end, intv):
             'label': '?broaderLabel'
         },
         'count': '?count'
-    }
+    },
+    'recon_provided_person_v1.sparql': {
+        'id': '?id',
+        'score': '?score',
+        'label': '?label'
+    },    
+    'recon_crm_v1.sparql': {
+        'id': '?id',
+        'score': '?score',
+        'label': '?label'
+    },        
 }
 
+RECON_MAX_BATCH_SIZE = 50
+@app.get('/recon', 
+    tags=['Reconciliation'],)
+async def recon_manifest():
+    return {
+        "versions": ["0.1"],
+        "name": "InTaVia",
+        "identifierSpace": "http://www.intavia.eu/idm-core/",
+        "schemaSpace": "http://www.intavia.eu/idm-core/Provided_Item",
+        "batchSize": RECON_MAX_BATCH_SIZE,
+        "defaultTypes": [
+            {
+                "id": "Person",
+                "name": "Person"
+            },
+            {
+                "id": "Group",
+                "name": "Group"
+            },
+            {
+                "id": "Place",
+                "name": "Place"
+            },
+        ]
+    }
+
+@app.post('/recon/reconcile',
+    response_model=ReconResponse,
```

Comment:
there seems to be something wrong with the response model. at least in my local instance the swagger definitone does not show a proper response shape